x-pack/plugins/maps/public/render_app.tsx

```diff
@@ -73,11 +73,11 @@ export async function renderApp({
     ...withNotifyOnErrors(getToasts()),
   });
 
+  const stateTransfer = getEmbeddableService().getStateTransfer();
+
   setAppChrome();
 
   function renderMapApp(routeProps: RouteComponentProps<{ savedMapId?: string }>) {
-    const stateTransfer = getEmbeddableService()?.getStateTransfer();
-
     const { embeddableId, originatingApp, valueInput } =
       stateTransfer?.getIncomingEditorState() || {};
```

Comment:
Can stateTransfer be undefined? Looks like you removed `?` from `getEmbeddableService().getStateTransfer()`. If stateTransfer can not be undefined can remove the `?` from line 82 as well.

Also, if stateTransfer is always defined, can you update https://github.com/elastic/kibana/blob/master/x-pack/plugins/maps/public/routes/map_page/saved_map/saved_map.ts#L58, making _stateTransfer required?